apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -1,9 +1,41 @@
-import React from 'react';
-
-/**
- * This is a placeholder for the upcoming "Review" tab in Javalab.
- * See related task: https://codedotorg.atlassian.net/browse/CSA-361
- */
-export default function ReviewTab() {
-  return <p>Review...</p>;
+import React, {Component} from 'react';
+import javalabMsg from '@cdo/javalab/locale';
+
+export default class ReviewTab extends Component {
+  state = {
+    readyForReview: false
+  };
```

Comment:
Will this be driven by Redux? Or I guess that's something we can revisit when more of the feature is in?

Reply:
mmm I hadn't really considered yet...I'm not sure to what degree whether a project is ready to review will be used by other parts of Javalab (ie, whether we'd need to keep it in Redux). I think we can wait and see until we build this out a little more.